locales/it/index.yml

```diff
@@ -237,6 +237,8 @@ wallet:
     newMethod: Scegli un altro metodo di pagamento
     addDescription: Puoi aggiungere i tuoi metodi di pagamento (carte, conti correnti,
       ecc.) per non doverli inserire ad ogni transazione
+    successful: Portafoglio aggiunto con successo
```

Comment:
yea this doesn't make much sense to the user :)

Reply:
yep definitely, that was pretty much a placeholder term, hopefully someone at some point will make sure all the messages make sense. For now, I'm changing these to "card"